tests/network/job.go

```diff
@@ -0,0 +1,51 @@
+package network
```

Comment:
Missing header license.

But I am not clear why this is not included under `libnet/job` as well.

Reply:
lets just merge the two whole job.go into the one that is on libnet/job ?
just split them because https://github.com/kubevirt/kubevirt/pull/10098#discussion_r1262404747 
but after doing that, it make more sense to merge them to one

Reply:
I think it makes sense to merge them, both serve as helpers.